notebooks/Media-Pretrained/03_Inference_Layer.ipynb

```diff
@@ -1018,6 +1215,34 @@
     "    \n"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "max_time = time.time() + 3*60*60 # 3 hours\n",
```

Comment:
There are two fairly large wait loops in this notebook (search "max_time = time").  They should both have a markdown comment to explain that we are going to wait for resources to be created before moving on.